pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -34,6 +34,11 @@ message KeyValue {
 	required string value = 2;
 }
 
+message KeyByteValue {
```

Comment:
`string` and `bytes` types are effectively identical. We could use the `KeyValue` message type here.

Reply:
I see. One of the recommendation is to use byte array when keys are stored. Since it's encrypted key, we should be ok with string. Will change it.

Reply:
Yes, in both cases you get the `ByteString` which is essentially a byte array

Reply:
I don't think you need to used base64. In protobuf wire protocol, `bytes` and `string` are exactly the same type and they're also exposed in the same way in the Java API through `ByteString` class. `ByteString` can hold both.

Reply:
You are right, ByteString can hold both. However, in our proto, the the "value" in KeyValue is defined as string, which can't hold ByteString.

In this use case, the result of encrypted key is a byte array which can be easily converted to ByteString without additional bytes. However, converting to String would require additional bytes.

Reply:
Actually, just checked with Andrews and the type is different in Java (`string` --> `String` and `bytes` --> `ByteString`)

Reply:
Thanks Matteo. Let us keep the KeyByteValue since it's efficient.